File: src/main/resources/static/js/question.js
$(function () {
    $(".questionMsg").hide();
    $("#question").submit(function () {
        if($("#title").val().trim() == "") {
            $(".questionMsg").show().html("标题不能为空！");
        } else if($("#description").val().trim() == "") {
            $(".questionMsg").show().html("问题描述不能为空！");
        } else if($("#tag").val().trim() == "") {
            $(".questionMsg").show().html("标签不能为空！");
        } else {
            $(".questionMsg").hide();
            return true;
        }
        return false;
    });
});

File: src/main/java/cn/lxg/community/model/Question.java
package cn.lxg.community.model;

import lombok.Data;

import java.io.Serializable;

/**
 * (Question)实体类
 */
@Data
public class Question implements Serializable {
    private static final long serialVersionUID = -29311715351876455L;
    
    private Integer id;
    
    private String title;
    
    private String description;
    
    private Long gmtCreate;
    
    private Long gmtModified;
    
    private Integer creator;
    
    private Integer commentCount;
    
    private Integer viewCount;
    
    private Integer likeCount;
    
    private String tag;


}